src/explore-education-statistics-frontend/src/modules/table-tool/components/FiltersForm.tsx

```diff
@@ -52,6 +52,16 @@ const FiltersForm = (props: Props & InjectedWizardProps) => {
     <WizardStepHeading {...props}>Choose your filters</WizardStepHeading>
   );
 
+  function getSectionError(errorLabel: string, spliceOut: string) {
```

Comment:
Why are we doing this? GDS error message guidance states that the error messages must be the same between the error summary and the input error.

https://design-system.service.gov.uk/components/error-summary/#how-it-works